DESCRIPTION

```diff
@@ -38,11 +38,15 @@ Imports:
     renv,
     rlang,
     rmarkdown,
+    tidyr,
     tibble,
     withr
 Suggests: 
-    testthat
+    knitr,
+    testthat (>= 3.0.0)
 Encoding: UTF-8
 LazyData: true
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.1.2
+Config/testthat/edition: 3
```

Comment:
there are existing tests.... are you certain that switching to version 3 of testthat will not cause problems?

Reply:
All automated tests (i.e. those that are in the `tests/testthat/` directory) are passing, as can be seen in the R CMD Checks on CI (I also manually run all tests before each push). 

Do you expect other problems? In particular, are there tests that use `testthat` that aren't run automatically when executing `ctrl + shift + T`? Or, do you expect problems that wouldn't be exposed by the test failing?

Reply:
just concerned/curious because last time I experimented with "edition 3" it broke a bunch of my tests so I just reverted

Reply:
Ok, and by "broke tests", you mean they were failing after? Since they are not failing here, do you think it's safe to assume we are good?

Reply:
probably... I'm pretty sure it caused some of my tests to error out.... I think it was in the data_prep repo though

Reply:
I'm a little worried because I haven't fully wrapped my head around the edition 3 changes yet, so I fear it may hamper my ability to write tests, but I guess I'll need to get there at some point anyway